recipes/odl/meta.yaml

```diff
@@ -0,0 +1,59 @@
+package:
+    name: odl
+    version: "0.8.0"
+
+source:
+    url: https://github.com/odlgroup/odl/archive/refs/tags/v0.8.0.tar.gz
+    sha256: dec092ce1f152707d821146046d4d1441672ad6c22d28dd46e1f77d79c210a38
+
+build:
+    number: 0
+    script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+    build:
+        - python
+        - setuptools >=65.6
+
+    host:
+        - python
+        - setuptools
+        - nomkl  # [not win]
+        - future >=0.16
+        - numpy >=1.19, <1.27
+        - scipy >=1.1
+        - packaging >=17.0
```

Comment:
All these dependencies should not be needed in host

Reply:
Removed in [5d684f8](https://github.com/conda-forge/staged-recipes/pull/27504/commits/5d684f894ae7311cfe8028e99cf520d6df648bb9)